### DISCUSSION TITLE
Consider adding Apious/Ads_Filter_Contents into badlists.txt.

### DISCUSSION THREAD

**Discussion by piquark6046** [uBlockOrigin/uAssets#20627] [General] [answered]
Consider adding [Apious/Ads_Filter_Contents](https://github.com/Apious/Ads_Filter_Contents/blob/1st/Ads_Filter_Contents.txt) into [badlists.txt](https://github.com/uBlockOrigin/uAssets/blob/master/filters/badlists.txt).

The misinformation is being posted in [their blog](https://apious.tistory.com/).
For example, the following methods are recommended by them in [the post](https://apious.tistory.com/422) for uBO's common configuration:

- Unsubscribing all uAssets filters lists
- Subscribing AdGuard filters lists (Base and Mobile)
- Subscribing their own filters list containing `youtube.com##+js(json-prune, playerResponse.adPlacements playerResponse.playerAds playerResponse.adSlots adPlacements playerAds adSlots legacyImportant)`.
- Subscribing List-KR filters list using   https://cdn.jsdelivr.net/gh/List-KR/List-KR@master/filter-AdGuard.txt.

And, they recommend their own filters list and AdGuard Base filters list to resolve YouTube anti-adblock.
Finally, count of the total visitors about the blog is 209 993.

https://listauthorschat.slack.com/archives/C05G8V3NERH/p1699789738997719

**Comment by Yuki2718** [comment, marked as answer]
https://github.com/uBlockOrigin/uAssets/commit/9112ec707b4bf0a959c16079b483981eb081fcf9
https://github.com/uBlockOrigin/uAssets/commit/5261782706c93b433eee37da597d819c31121ac0

**Comment by piquark6046** [comment]
@Yuki2718 Please add `https://raw.githubusercontent.com/Apious/Ads_Filter_Contents/1st/Ads_Filter_Contents.txt?_=5` too.

<details><summary>Screenshot</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/98787049/034fa2be-7bcf-48bc-b9f6-49da75af9d53)

</details>

**Reply by Yuki2718** [reply]
https://github.com/uBlockOrigin/uAssets/commit/951d557c19f94e2e23aefa750f9e8bbd2d6a20aa

**Reply by ghost** [reply]
This number changes every hour when forcing manual update.

***

uBO will match the original URL. No need to add the version with `_=X` unless it's in the list subscription URL.

**Reply by Yuki2718** [reply]
Okay, reverted.

**Reply by piquark6046** [reply]
https://github.com/Apious/Ads_Filter_Contents/commits/1st/Ads_Filter_Contents_.txt

Please block this too.

**Reply by Yuki2718** [reply]
So the author added `_` at the tail? If he keeps trying to circumvent, we won't waste time for that. Happened once for fuckadblock list.

**Reply by Yuki2718** [reply]
https://github.com/uBlockOrigin/uAssets/commit/ea090d6160ef16c2ee575f5554c25db5abe6eed3

**Reply by piquark6046** [reply]
Then, can i get the write permission for uAssets?

**Reply by Yuki2718** [reply]
Will discuss. But the concept of badlists is not on fully preventing user to subscribe the list, circumvention is intentionally allowed.

**Reply by ghost** [reply]
Don't block it until it starts popping up in user reports.

**Reply by Apious** [reply]
Hello There
would you please tell me of block ads_filter_contents?
i think. It seems like I broken the rule and got blocked.
If you tell me the rules, I will follow them.
thx.

**Reply by piquark6046** [reply]
I don't understand your meaning correctly.
Please use a translator if you cannot write English well.

**Reply by gorhill** [reply]
@Apious You tell people to disable all of uBlock filters -- this is wrong. The file you tell your visitors to use to configure uBO has this:

```json
  "selectedFilterLists": [
    "user-filters",
    "adguard-generic",
    "adguard-mobile",
    "easylist",
    "easyprivacy",
    "KOR-1",
    "https://raw.githubusercontent.com/Apious/Ads_Filter_Contents/1st/Ads_Block_Contents.txt",
    "https://raw.githubusercontent.com/Apious/Ads_Filter_DNS/1st/Ads_Block_DNS.txt",
    "https://cdn.jsdelivr.net/gh/adguardteam/HostlistsRegistry@main/assets/filter_25.txt"
  ],
  ...
  "whitelist": [
    "# chrome-extension-scheme",
    "# chrome-scheme",
    "# edge-scheme",
    "about-scheme",
    "file-scheme",
    "github.com",
    "kr.battle.net",
    "moz-extension-scheme",
    "opera-scheme",
    "spam.kisa.or.kr",
    "teams.microsoft.com",
    "vivaldi-scheme",
    "www.aliexpress.com",
    "www.hometax.go.kr",
    "www.kics.go.kr",
    "www.notion.so",
    "www.tistory.com",
    "wyciwyg-scheme"
  ],
  "dynamicFilteringString": "* coocha.co.kr * allow\n* daum.net * allow\n* daumcdn.net * allow\n* dmm.co.jp * allow\n* kakaocdn.net * allow\n* mediacategory.com * block\n* phonearena.com * allow\n* analytics.ad.daum.net * block\n* dmca.com * allow\n* dood.video * allow\n* scv.band.us * allow\n* internetdownloadmanager.com * block"
```

Here we call this misconfiguring uBO and we are on the receiving end of uBO not working as expected for people following your advice.
